internal/app/server/repository/file_storage.go

```diff
@@ -0,0 +1,146 @@
+package repository
+
+import (
+	"encoding/json"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+type FileBackedStorage struct {
+	MemStorage
+	filePath      string
+	storeInterval time.Duration
+	stopChan      chan struct{}
+	storeMutex    sync.Mutex
+	logger        *logrus.Logger
+}
+
+type MetricsData struct {
```

Comment:
[CHANGE REQUEST] It should be in the model package.

Reply:
Done